src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VepAnnotationFileWriter.java

```diff
@@ -68,7 +75,9 @@ public void update(ExecutionContext executionContext) throws ItemStreamException
 
     @Override
     public void close() throws ItemStreamException {
-        vepProcess.close();
+        if (vepProcessOpened) {
```

Comment:
Oh, you meant that the VepProcess takes care of the double close, so this can be a direct call. yes, that's true, I'll change that back.